schemas/context/segmentmembership.schema.json

```diff
@@ -29,6 +29,42 @@
           "format": "date-time",
           "description": "The timestamp when the assertion of segment membership was made."
         },
+        "xdm:previousQualificationTimes": {
```

Comment:
Will drop this field, if you need to show previous states of segment membership , just repeat the segment membership definition in the array.

Don't need this on experience event either.